docs/data/base/components/popper/UnstyledPopperBasic/css/index.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import Popper from '@mui/base/Popper';
+import { useTheme } from '@mui/system';
+
+export default function SimplePopper() {
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(anchorEl ? null : event.currentTarget);
+  };
+
+  const open = Boolean(anchorEl);
+  const id = open ? 'simple-popper' : undefined;
+
+  return (
+    <React.Fragment>
+      <button aria-describedby={id} className="Button" onClick={handleClick}>
```

Comment:
```suggestion
      <button type="button" aria-describedby={id} className="Button" onClick={handleClick}>
```

In order to fix the lint issue. Also, different browsers may use different default type value, so just to be on the safe side :)